src/resolvers/user/resolver.ts

```
@@ -174,9 +190,178 @@ export class UserResolver {
+                                                        Password</a>
+                                                </td>
+                                            </tr>
+                                            <tr>
+                                                <td style="height:40px;">&nbsp;</td>
+                                            </tr>
+                                        </table>
+                                    </td>
+                                <tr>
+                                    <td style="height:20px;">&nbsp;</td>
+                                </tr>
+                                <tr>
+                                    <td style="text-align:center;">
+                                        <p style="font-size:14px; color:rgba(69, 80, 86, 0.7411764705882353); line-height:18px; margin:0 0 0;">&copy; <strong></strong></p>
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td style="height:80px;">&nbsp;</td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+                </table>
+                <!--/100% body table-->
+            </body>
+
+            </html>
+        `;
+    await sendEmail(
+      "no-reply@selfieguess.io",
+      user.email,
+      "Reset Password",
+      html
+    );
+  }
+
+  //Forget Password
+  @Mutation(() => String!)
+  async forgetPassword(@Arg("email") email: string) {
+    const user = await User.findOne({ where: { email } });
+    if (user) {
+      const check = await this.reload(user, Action.RESET_PASSWORD);
+      await this.sendForgetPasswordEmail(user, check);
+    }
+    return "Please check your email. If you are registered with us, you'll recieve an email with instructions to reset your password.";
```

should throw an error when the user doesn't exist.

---

fixed this.